plugins/rapture/nexus-rapture-plugin/src/main/java/org/sonatype/nexus/rapture/internal/security/SessionAuthenticationFilter.java

```diff
@@ -52,6 +52,19 @@
 
   public static final String P_REMEMBER_ME = "rememberMe";
 
+  public static final String DELETE_METHOD = "DELETE";
+
+  /**
+   * Allow access, by default rules, and also if this is a request to logout.
+   */
+  @Override
+  protected boolean isAccessAllowed(final ServletRequest request,
+                                    final ServletResponse response,
+                                    final Object mappedValue)
+  {
+    return super.isAccessAllowed(request, response, mappedValue) || isLogoutRequest(request, response);
```

Comment:
@kellyrob99 okay, I'll have to look, I think its a rats nest of super calls up the chain and basically pointless doing a bunch of stupid junk, so it probably could/should just be specific to allow access to DELETE.